scripts/gen_priv_stacks.py

```diff
@@ -101,10 +101,14 @@ def parse_args():
 def main():
     parse_args()
 
+    assert args.kernel, "--kernel ELF required for --gperf-output"
```

Comment:
There's a `required=True` on the `parser.add_argument()` that adds `--kernel`, so is this needed?

Reply:
You're right, I saw the mostly duplicated `main()` so I blindly copied the assert from the other script. Will remove.